edx2course_axis.py

```diff
@@ -51,113 +56,121 @@
 DATADIR = "DATA"
 
 VERBOSE_WARNINGS = True
-#FORCE_NO_HIDE = True
 FORCE_NO_HIDE = False
 
-# <codecell>
-
-#-----------------------------------------------------------------------------
+LOGGER = logging.getLogger()
```

Comment:
I don't want to stray from convention, so we'll do whatever the standard is.

My thinking is that there are three options:

* Use an all-caps name.
* Disable "bad name" globally.
* Instantiate loggers within each scope (class and function).

I think the second is bad, the third is messy (especially if the loggers are logging to the same resources, such as files), and the first is kind of ugly and an abuse of the word "constant."

I'll defer to the MITODL standard on this one. So, change it to lower-case and globally disable bad variable names?